README.md

```diff
@@ -95,6 +95,13 @@ The password argument is optional. It's required only if your system is unable t
 
 ```python install_all.py --password xyz```
 
+
+## Docker environment execution
+
+For instructions on how to compile and execute Snaskii21 using Docker, check the [Docker README](/docker_env/README.md).
+
+
+
```

Comment:
Better to remove these extra lines.

Reply:
Since you approved these changes this week, I believe this conversation can be resolved. Feel free to revert the resolution otherwise.